engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/product-repositories.controller.js

```diff
@@ -49,8 +49,8 @@ angular.module('Bastion.products').controller('ProductRepositoriesController',
         $scope.errorMessages = [];
 
         $scope.checksums = [{name: translate('Default'), id: null}, {id: 'sha256', name: 'sha256'}, {id: 'sha1', name: 'sha1'}];
-        $scope.repositoriesTable = repositoriesNutupane.table;
-        $scope.repositoriesTable.removeRow = repositoriesNutupane.removeRow;
+        $scope.detailsTable = repositoriesNutupane.table;
```

Comment:
I haven't met any errors even without this line. Could you please check it again with the latest commit if you still see any errors?